server/aws-lsp-codewhisperer/src/language-server/inline-completion/codewhispererController.ts

```diff
@@ -0,0 +1,128 @@
+import {
+    CancellationToken,
+    CredentialsProvider,
+    InitializeParams,
+    InlineCompletionListWithReferences,
+    InlineCompletionWithReferencesParams,
+    Logging,
+    LogInlineCompletionSessionResultsParams,
+    Telemetry,
+    Workspace,
+} from '@aws/language-server-runtimes/server-interface'
+import { AmazonQBaseServiceManager } from '../../shared/amazonQServiceManager/BaseAmazonQServiceManager'
+import { TelemetryService } from '../../shared/telemetry/telemetryService'
+import { UserWrittenCodeTracker } from '../../shared/userWrittenCodeTracker'
+import { CodePercentageTracker } from './codePercentage'
+import { DocumentChangedListener } from './documentChangedListener'
+import { SessionManager } from './session/sessionManager'
+import { RecentEditTracker } from './tracker/codeEditTracker'
+import { CursorTracker } from './tracker/cursorTracker'
+import { EditCompletionHandler } from './editCompletionHandler'
+import { InlineCompletionHandler } from './inlineCompletionHandler'
+import { LogInlineCompletionSessionResultsHandler } from './logInlineCompletionSessionResultsHandler'
+import { RejectedEditTracker } from './tracker/rejectedEditTracker'
+import { AcceptedInlineSuggestionEntry, CodeDiffTracker } from './codeDiffTracker'
+
+export class CodeWhispererController {
+    // API handlers
+    editCompletionHandler: EditCompletionHandler | undefined
+    private inlineCompletionHandler: InlineCompletionHandler | undefined
+    private logInlineCompletionSessionResultsHandler: LogInlineCompletionSessionResultsHandler | undefined
+
+    constructor(
+        readonly sessionManager: SessionManager,
+        readonly editsSessionManager: SessionManager
+    ) {}
+
+    async init(
```

Comment:
can you move this init to the constructor so that `inlineCompletionHandler` does not need to be undefined?